bitcoin-submittx

```diff
@@ -23,142 +24,160 @@ from bitcoin.core.serialize import Hash
 from bitcoin.messages import msg_version, msg_inv, msg_ping, msg_verack, msg_pong, msg_tx, messagemap, MsgSerializable, MSG_TX, MSG_BLOCK
 from bitcoin.net import CInv
 
+from random import shuffle
+
+# needed for QR code reading
```

Comment:
The cv2 dependency probably should be optional. It's a huge library to require if the user doesn't need QR code / camera support.

Reply:
I was basically trying to keep install commands as simple as possible. However, I was also thinking about your concern but needed to figure out how to do optional dependencies with pip and wanted to just publish it as is to see if anyone cared about these upgrades before cleaning up this part.